src/ansys/fluent/core/launcher/launcher_utils.py

```diff
@@ -282,28 +383,18 @@ def _get_mode(mode: Optional[Union[FluentMode, str, None]] = None):
     return mode
 
 
-def _raise_exception_g_gu_in_windows_os(additional_arguments: str) -> None:
-    """If -g or -gu is passed in Windows OS, the exception should be raised."""
-    additional_arg_list = additional_arguments.split()
-    if _is_windows() and (
-        ("-g" in additional_arg_list) or ("-gu" in additional_arg_list)
+def _raise_non_gui_exception_in_windows(
+    exposure: FluentExposure, product_version: FluentVersion
+) -> None:
+    """Exposure < hidden_gui is not supported in Windows in Fluent version < 24.1."""
```

Comment:
Exception message should be in plain English with tagged code entities and the less than operator written out.